pornbothunter.py

```diff
@@ -239,18 +214,25 @@ def publish_summary_tweet():
         print e
 
 
+def init():
+    auth = tweepy.OAuthHandler(data["consumer_key"], data["consumer_secret"])
+    auth.set_access_token(data["access_token"], data["access_secret"])
+    api = tweepy.API(auth)
+    return data["patterns"]
+
 if __name__ == '__main__':
+    patterns = init();
     while True:
-        result = GoogleSearch().search(random.choice(patterns), num_results=100)
+        result = GoogleSearch().search("site:twitter.com \"" + random.choice(patterns) + "\"", num_results=100)
```

Comment:
By using single quotes, you don't need to escape the double quotes. By using `"".format()`, you don't need string concatenation via `+`.